docs/reference/errors-and-warnings/NU1110.md

```diff
@@ -0,0 +1,33 @@
+---
+title: NuGet Error NU1110
+description: NU1110 error code
+author: donnie-msft
+ms.author: eagoodso
+ms.date: 10/27/2023
+ms.topic: reference
+f1_keywords: 
+  - "NU1110"
+---
+
+# NuGet Error NU1110
+
+<pre>NU1110: The package `B` is available in the Global packages folder, but the source it came from `https://api.nuget.org/v3/index.json` is not one of the configured sources.
+</pre>
+
+### Issue
+
+A previous restore retrieved package `B` from a package source which your current solution does not have configured.
+
+While performing a package installation of package `A`, a dependent package `B` was needed transitively to complete the action. Restore found `B` in your machine's [Global packages folder](https://aka.ms/troubleshoot_nuget_cache), and the accompanying `nupkg.metadata` indicates a `Source` value which it cannot find in your solution.
```

Comment:
```suggestion
While performing a package installation of package `A`, a dependent package `B` was needed transitively to complete the action. Restore found `B` in your [Global packages folder](../../consume-packages/managing-the-global-packages-and-cache-folders), and the accompanying `nupkg.metadata` indicates a `Source` value which it cannot find in your solution.
```

The global packages folder is user specific. I think we don't need extra specificity. 

Also the guidance for referencing internal docs is to reference the location internally.

Reply:
Done 👍